pkg/tsdb/influxdb/flux/builder.go

```diff
@@ -129,20 +128,29 @@ func (fb *FrameBuilder) Append(record *query.FluxRecord) error {
 		}
 
 		if fb.isTimeSeries {
+			frameName := ""
+			name := record.ValueByKey("_measurement")
+			if name != nil {
+				frameName = name.(string)
```

Comment:
Since this is a core datasource, I think it is better to be a defensive on any type assertion.s therefore this should use the same `ok` pattern as earlier.